lib/boku/client.py

```diff
@@ -69,8 +71,8 @@ def _sign(self, params):
         params['sig'] = get_boku_request_signature(self.secret_key, params)
         return params
 
-    def api_call(self, path, params):
-        if 'timestamp' not in params:
+    def api_call(self, path, params, add_timestamp=True):
+        if add_timestamp and 'timestamp' not in params:
             params['timestamp'] = str(calendar.timegm(time.gmtime()))
```

Comment:
could also be:

    if add_timestamp:
        params.setdefault('timestamp', str(calendar.timegm(time.gmtime())))